src/katgpucbf/fgpu/pfb.py

```diff
@@ -40,25 +41,41 @@ class PFBFIRTemplate:
         The number of taps that you want the resulting PFB-FIRs to have.
     channels
         Number of channels into which the input data will be decomposed.
-    dig_sample_bits
-        Bits per digitiser sample.
+    input_sample_bits
+        Bits per input sample. If `complex_input` is true, the input values
+        are floating-point complex numbers and this is the number of bits in
+        the complex value (and must equal 64). Otherwise, the inputs are
+        packed integers, and the value must be in
+        :data:`DIG_SAMPLE_BITS_VALID`.
     unzip_factor
         The output is reordered so that every unzip_factor'ith pair of
         outputs is placed contiguously.
     """
 
     def __init__(
-        self, context: AbstractContext, taps: int, channels: int, dig_sample_bits: int, unzip_factor: int = 1
+        self,
+        context: AbstractContext,
+        taps: int,
+        channels: int,
+        input_sample_bits: int,
+        unzip_factor: int = 1,
+        *,
+        complex_input: bool = False,
```

Comment:
I know it's about as obvious as a parameter can get, but do you think it's worth having a oneliner under Parameters in the docstring?